outputs.tf

```diff
@@ -2,6 +2,14 @@ output "ansible_control_access_ip_v4" {
   value = openstack_compute_instance_v2.ansible_control.access_ip_v4
 }
 
+output "seed_access_ip_v4" {
+  value = openstack_compute_instance_v2.seed.access_ip_v4
+}
```

Comment:
I don't see this one being used.

Reply:
This was used for convenience when accessing the seed node or more importantly the public network of the deployed openstack environment. Particularly useful for testing DNS responses as sshuttle can proxy DNS queries.

```sshuttle -r $(terraform output -raw ssh_user)@$(terraform output -raw seed_access_ip_v4) 192.168.39.0/24 --dns --to-ns 192.168.39.4```

Rather than remove this output I have expanded the `README.rst` to include information on how to use it.